tests/management/commands/test_poll_providers.py

```diff
@@ -304,6 +305,8 @@ def test_poll_provider_v0_4_realtime(client, requests_mock):
     # Note: testing with the default "POLLER_CREATE_REGISTER_EVENTS = False"
     # This time we didn't poll long ago
     last_event_time_polled = timezone.now() - datetime.timedelta(3)
+    # MDS precision is to the millisecond only
+    last_event_time_polled = last_event_time_polled.replace(microsecond=1000)
```

Comment:
je ne comprends pas bien le commentaire :) ça veut dire quoi ?

Reply:
:gb:  The test would fail because of a microsecond difference.